apps/emqx_cluster_link/src/emqx_cluster_link_schema.erl

```diff
@@ -137,17 +137,22 @@ link_name(#{name := Name}) -> Name;
 link_name(#{<<"name">> := Name}) -> Name.
 
 topics_validator(Topics) ->
```

Comment:
maybe split the impl, one function to check duplication, another to validate topics.